raisinbread/RB_files/RB_Config.sql

```diff
@@ -91,5 +91,6 @@ INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (94,99,'true');
 INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (95,49,'1000');
 INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (96,100,'environment');
 INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (97,70,'/data-raisinbread/');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (99,101,'Y-m-d');
```

Comment:
im wondering why its ID 99 and not 98

Reply:
```suggestion
INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (98,101,'Y-m-d');
```

I'm just gonna change it manually.